R/dribble.R

```diff
@@ -85,7 +85,7 @@ maybe_dribble <- function(x) {
   if (all(ok)) {
     new_dribble(x)
   } else {
-    x
+    structure(x, class = class(tibble::tibble()))
```

Comment:
This little exercise revealed that some dplyr verbs don't strip the dribble class, even when they should! Hence this. Is this the best way to get and apply the class attribute for a plain vanilla tibble? Or should I write `as_tibble.dribble()` 🙄?